src/lib/stategraph.rs

```diff
@@ -58,143 +55,7 @@ use grammar::{PIdx, Grammar, Symbol, SIdx};
 //   Measuring and extending LR(1) parser generation
 //     Xin Chen, PhD thesis, University of Hawaii, 2009
 
-
-/// The type of "context" (also known as "lookaheads")
-pub type Ctx = BitVec;
-
-#[derive(Clone, Debug, PartialEq)]
-pub struct Itemset {
-    pub items: HashMap<(PIdx, SIdx), Ctx, BuildHasherDefault<FnvHasher>>
-}
-
 impl Itemset {
```

Comment:
Okay, I've now seen the next commit where you move `stategraph.rs` to `pager.rs`, so everything makes sense.